addons/event/tests/common.py

```diff
@@ -93,6 +93,7 @@ def setUpClass(cls):
         })
 
         # set country in order to format Belgian numbers
+        _ =  cls.user_employee.company_id.state_id  # noqa -- to check with RCO - need prefetch here
```

Comment:
@rco-odoo 

for a strange reason, if I dont prefetch here, the writ eon reS.company si wrongly done with False a value.

Not a critical bug for this PR since you don't change every day you company details without the Ux form, but maybe investigate this one will fix others unknown situations ;) 

thx